internal/cmd/egctl/experimental.go

```diff
@@ -26,6 +26,8 @@ func newExperimentalCommand() *cobra.Command {
 	experimentalCommand.AddCommand(newStatsCommand())
 	experimentalCommand.AddCommand(newStatusCommand())
 	experimentalCommand.AddCommand(newDashboardCommand())
+	experimentalCommand.AddCommand(newInstallCommand())
```

Comment:
hi @ShyunnY, the code looks good! 

since we have decided to set these two subcmds as experimental, can we add a section `egctl experimental install & uninstall` in https://gateway.envoyproxy.io/latest/user/operations/egctl/ to describe how to use these cmds.

and also, since we can install EG using egctl, maybe this doc https://gateway.envoyproxy.io/latest/install/install-egctl/ can be renamed as `install-with-egctl`, and how to install egctl will be a section in it. WDYT?

Reply:
IMO, I think https://gateway.envoyproxy.io/latest/install/install-egctl/ does not need to be changed, this part of the doc is meant to tell us how to install egctl, we are not supposed to add other information here. Currently install/uninstall is only in experimental status, so it is enough for us to just add description information at https://gateway.envoyproxy.io/latest/user/operations/egctl/.

When we later consider changing install/uninstall to stable, we will consider adding additional docs to specifically describe how to use egctl to install or uninstall eg.

I think this is more appropriate, what do you think? :)

Reply:
SGTM